test/sequential/test-net-connect-local-error.js

```diff
@@ -6,20 +6,39 @@ const net = require('net');
 // EADDRINUSE is expected to occur on FreeBSD
 // Ref: https://github.com/nodejs/node/issues/13055
 const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
-const client = net.connect({
+
+const optionsIPv4 = {
   port: common.PORT,
   localPort: common.PORT + 1,
   localAddress: common.localhostIPv4
-});
+};
+
+const optionsIPv6 = {
+  host: common.localhostIPv6,
+  port: common.PORT + 2,
+  localPort: common.PORT + 3,
+  localAddress: common.localhostIPv6
+};
 
-client.on('error', common.mustCall(function onError(err) {
+const onError = (err, options) => {
```

Comment:
Could this be a function declaration?
```js
function onError(err, options) {
```

Reply:
It's a named function expression.

Reply:
It's a style preference, function declaration are much more common in the test suite than named function expressions.